src/types/HeaderTypes.ts

```diff
@@ -0,0 +1,68 @@
+// TODO - share with banner/epic!
```

Comment:
would this be a lot of work?

Reply:
I wrote a similar comment for the banner types - https://github.com/guardian/support-dotcom-components/blob/main/src/types/BannerTypes.ts#L6
so I think it's a separate piece of work